USER_GUIDE.md

```diff
@@ -303,3 +303,9 @@ var client = new OpenSearchLowLevelClient(config);
 ```
 
 Note the main difference here is that we are instantiating an `OpenSearchLowLevelClient` rather than `OpenSearchClient`, and `ConnectionConfiguration` instead of `ConnectionSettings`.
+
+
+## Advanced Features
+
+- [Making Raw JSON Requests](guides/json.md)
+  - [Sample Code](/samples/Samples/Program.cs)
```

Comment:
You do not need the `[Sample Code]` line here. Additionally there should be a entry added to the table of contents at the top of this file for the `Advanced Features`